_summaries/maas2013rectifier_2.md

```diff
@@ -1,19 +1,36 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
+title: Rectifier Nonlinearities Improve Neural Network Acoustic Models, Maas, Hannun, Ng; 2013
+paper: 	{maas2013rectifier}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: sritank
+score: 8/10
 ---
 
 TODO: Summarize the paper:
 * What is the core idea?
+
+DNNs with ReLU and other non-sigmoid activation functions performed better on image recognition tasks. So the paper explores using DNNs as acoustic models and how rectifier non-linearities can improve estimation accuracy of the acoustic model.
+
 * How is it realized (technically)?
+
+The authors train a DNN acoustic model in an LVCSR (large vocabulary continuous speech recognition) task. To study the impact of different non-linearities on the training accuracy, they do not use regularization or pre-trained weights for the network. 
+
+For the same speech recognition task, they replace the GMM/HMM acoustic model with a DNN. The DNN training inputs are MFCC (Mel frequency cepstral coefficients) features of the audio signal. The output of the DNN is the senone likelihood probability function. They train DNNs (2-4 layers containing 2048 units/layer) with sigmoid, ReLU and Leaky ReLU activation functions using stochastic gradient descent. 
```

Comment:
I think it is a good idea to mention the loss function that is used (Cross Entropy) and the batch size of 256. I would also mention that they tried different learning rates in the set [0.1, 0.05, 0.01, 0.005, 0.001]. I found it interesting that they started with a momentum of 0.5 and increased it 0.9 after a certain number of SGD iterations, which might be worth adding to your summary as well.

Reply:
I added the point about the loss function. Regarding the others, they never mention if the learning rates or momentum increased performance in any way. The network would probably just take more iterations to converge if momentum was constant. But the performance of the final trained network is unchanged. So I don't think it's necessary to mention about the learning rate and momentum

Reply:
They mentioned that 0.01 led to the fastest convergence, but I could see how that is not crucial to the summary. On second thought, stating the momentum is likely not useful here either.